game/src/common/minimap.rs

```diff
@@ -99,11 +99,11 @@ impl MinimapControls<App> for MinimapController {
 fn make_tool_panel(ctx: &mut EventCtx, app: &App) -> Widget {
     let buttons = ctx
         .style()
-        .btn_solid_light()
+        .btn_solid_floating()
```

Comment:
One place where we continue to use both dark and light buttons in the theme is here. I've given it a more semantic name since it'll eventually be "light" in the night theme and "dark" in the day theme.

In night theme, a `btn_solid` is used within panels, and looks like this:
<img width="154" alt="Screen Shot 2021-02-23 at 9 39 36 AM" src="https://user-images.githubusercontent.com/217057/108891525-ac9d6200-75c3-11eb-8358-49adc590a649.png">

In night theme a `btn_solid_floating` is used outside of panels, like these three minimap controls. It's "free-floating"... hence the name. I'm not wild about the name if you can think of a better one.
<img width="102" alt="Screen Shot 2021-02-23 at 9 39 18 AM" src="https://user-images.githubusercontent.com/217057/108891526-ad35f880-75c3-11eb-8b0c-c2cb68a84aa0.png">

Reply:
I should also add that, though we currently use `btn_solid` all over the place, in the ultimate design vision it's used very sparingly.